pkg/cache/cache_test.go

```diff
@@ -413,17 +416,33 @@ func CacheTest(createCacheFunc func(config *rest.Config, opts cache.Options) (ca
 					Expect(out.Items).Should(HaveLen(1))
 					Expect(out.Items[0].GetNamespace()).To(Equal(testNamespaceOne))
 
-					By("listing all namespaces - should still be able to get a cluster-scoped resource")
-					namespaceList := &unstructured.UnstructuredList{}
-					namespaceList.SetGroupVersionKind(schema.GroupVersionKind{
+					By("listing all nodes - should still be able to list a cluster-scoped resource")
+					nodeList := &kmetav1.PartialObjectMetadataList{}
```

Comment:
This needs to remain an unstructured.UnstructuredList, otherwise the two testcases are identical

Reply:
Good catch! That was an accidental copy/paste error.